test/updaters/fixtures/dotnet/SecurityCenter.V1.csproj

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Do we still do anything with `.csproj` files, in the strategy I only saw apis.json being updated.

Reply:
Do, but I somehow added the updater a long time ago without a test